packages/mirrorful/editor/src/components/ColorPalette/EditColorModal.tsx

```diff
@@ -81,21 +86,30 @@ export function EditColorModal({
         >
           <Flex flexDirection="column" flex="1">
             <FormControl>
-              <FormLabel>Variable Name</FormLabel>
-              <Input
-                placeholder="e.g. Pepsi Blue"
-                size="md"
-                value={name}
-                onChange={(e) => setName(e.target.value)}
-                onFocus={(e) => {
-                  setShowBaseColorPicker(true)
-                }}
-                onKeyPress={(event) => {
-                  if (event.key === 'Enter' && baseRef.current) {
-                    baseRef.current.focus()
-                  }
-                }}
-              />
+                <Flex flex="1">
+                  <FormLabel>Variable Name</FormLabel>
+                  <Tooltip
+                  placement="right"
+                  closeDelay={500}
+                  hasArrow
+                  label={"Variable names don't need a hyphen."}>
+                  <InfoIcon css={{ marginTop: '5px', marginLeft: '-6px' }} />
```

Comment:
Ideally, we shouldn't use negative margin, instead we should use marginRight.

Reply:
Hey @alexdanilowicz , yeah Ideally I would not have used negative margins. But I don't think that marginRight will work in this scenario due to the box structure of FormLabel field, it contains some inbuilt spacing and rightMargin won't overlap it. I can remove it all together but then the space between FormLabel and InfoIcon also increases. Had to use negative margin on purpose to reduce that spacing.

![Screenshot 2023-03-14 at 11 00 56 PM](https://user-images.githubusercontent.com/44923038/225089446-5d3bf137-4e8d-4da2-9183-a504d1c1fc04.png)

Reply:
oh that makes sense, thanks for teaching me something new and the detailed screenshot!